cmd/build/helmify/static/README.md

```diff
@@ -88,6 +88,7 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
 | disableMutation                               | Disable mutation                                                                                                                                                                                                                                    | `false`                                                                   |
 | validatingWebhookTimeoutSeconds               | The timeout for the validating webhook in seconds                                                                                                                                                                                                   | `3`                                                                       |
 | validatingWebhookFailurePolicy                | The failurePolicy for the validating webhook                                                                                                                                                                                                        | `Ignore`                                                                  |
+| validatingWebhookObjectSelector               | The label selector to further refine which namespaced resources will be selected by the webhook. Please note that an exemption label could mean some resources can circumvent all Gatekeeper validations.                                           | `{}`                                                                      |
```

Comment:
"Please note that an exemption label could mean some resources can circumvent all Gatekeeper validations." -> 

"Please note that an exemption label means users can circumvent Gatekeeper's validation webhook unless measures are taken to control how exemption labels can be set."

Reply:
👍  Done

Reply:
Also updated the mutatingWebhookObjectSelector description similarly

Reply:
Thanks!